src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -136,6 +136,16 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
                     })
                 .Where(item => item != null));
 
+            if (searchParams.Include != null)
+            {
+                searchExpressions.AddRange(searchParams.Include.Select(
```

Comment:
I wonder how :iterate is handled..

Reply:
Open for three years now! Also note that `:iterate` in R4 is `:recurse` in STU3.
Do we return an error when specified?